### PR TITLE
Update user role documentation

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for the issue #1657

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 39 sec

File: docs/src/user/manual/user-roles.md [hunk 1]
```diff
@@ -137,41 +121,41 @@ There are currently 10 roles:
...
+8. **National data scientist**: A national data reviewer who can view data down to the property level and use the data for e.g. planning and permitting.
 
     ![National data scientist](./img/user-role-8.png)
 
     * **Full access**: The national data scientist has full access to login, register, about, contact, profile, notification, reminders, two-factor authentication, and help page. National data scientists can access the documentation site.
 
     * **Access with conditions**: The national data scientist can access the following pages with conditions.
 
-        1. **Organisation member page**: Can only add organisation members if instated as a manager of an organisation/ their own organisation.
+        1. **Organisation member page**: Can only add organisation members if instated as a manager of an organisation or their organization.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/user-roles.md:138]:
```suggestion
        1. **Organisation member page**: Can only add organisation members if instated as a manager of an organisation or their organisation.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 26 sec

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 22 sec

**Pull request merged**